doc/code-generator/build.gradle

```diff
@@ -0,0 +1,138 @@
+  generatorName.set("python")
+  remoteInputSpec.set(inputURL)
+  outputDir.set(layout.buildDirectory.dir("python").get().getAsFile().getAbsolutePath())
+  apiPackage.set("com.wci.termhub.api")
+  invokerPackage.set("com.wci.termhub.api.invoker")
+  modelPackage.set("com.wci.termhub.api.models")
+  configOptions.set([
+      "packageName": "python-examples",
+      "hideGenerationTimestamp": "false",
+  ])
+}
+
+/**
+ * Build a Go client by default: https://openapi-generator.tech/docs/generators/go/
+ * To use a local file, replace {remoteInputSpec.set(inputURL)} with input {inputSpec.set(inputFile)}
+ */
+//tasks.register('buildGoSdk', GenerateTask) {
+//  generatorName.set("go")
+//  remoteInputSpec.set(inputURL)
+//  outputDir.set("$buildDir/go")
+//  apiPackage.set("com.wci.termhub.api")
+//  invokerPackage.set("com.wci.termhub.api.invoker")
+//  modelPackage.set("com.wci.termhub.api.models")
+//  configOptions.set([
+//      "packageName": "go-examples",
+//      "hideGenerationTimestamp": "false",
+//  ])
+//}
+
+/**
+ * Build a JS client by default: https://openapi-generator.tech/docs/generators/javascript/
+ * To use a local file, replace {remoteInputSpec.set(inputURL)} with input {inputSpec.set(inputFile)}
+ */
+//tasks.register('buildJavascriptSdk', GenerateTask) {
+//  generatorName.set("javascript")
+//  remoteInputSpec.set(inputURL)
+//  outputDir.set("$buildDir/javascript")
+//  apiPackage.set("com.wci.termhub.api")
+//  invokerPackage.set("com.wci.termhub.api.invoker")
+//  modelPackage.set("com.wci.termhub.api.models")
+//  configOptions.set([
+//      "packageName": "javascript-examples",
+//      "hideGenerationTimestamp": "false",
+//  ])
+//}
+
+/**
+ * Build a bash client by default: https://openapi-generator.tech/docs/generators/bash/
+ * To use a local file, replace {remoteInputSpec.set(inputURL)} with input {inputSpec.set(inputFile)}
+ */
+//tasks.register('buildBashSdk', GenerateTask) {
+//  generatorName.set("bash")
+//  remoteInputSpec.set(inputURL)
+//  outputDir.set("$buildDir/bash")
+//  apiPackage.set("com.wci.termhub.api")
+//  invokerPackage.set("com.wci.termhub.api.invoker")
+//  modelPackage.set("com.wci.termhub.api.models")
+//  configOptions.set([
+//      "packageName": "bash-examples",
+//      "hideGenerationTimestamp": false,
+//      "generateBashCompletion": true,
+//      "generateZshCompletion": true,
+//      "hostEnvironmentVariable":HOST='TODO',
+//  ])
+//}
+
+/** Runs all code-gen tasks for each language to generate the clients */
+tasks.register("buildClients") {
+  dependsOn("buildJavaSdk",
+      "buildPythonSdk",
+//            "buildGoSdk",
+//            "buildBashSdk",
+//            "buildJavascriptSdk"
+  )
+  doLast {
+    println 'All client sdk generated. They can be found in the code-gen/build folder.'
+  }
+}
+
+compileJava.dependsOn tasks.named("buildClients")
+build.dependsOn("buildClients")
+publishToMavenLocal.dependsOn = ["buildJavaSdk`",
```

Comment:
Why does publish to maven depend on a python code gen? It is not a python code repo

Reply:
would you say this is not necessary then? I'm not even sure we plan to publish any of the code-generate to a local version.